examples/index.json

```diff
@@ -3701,15 +3701,15 @@
       {
         "name": "Inbound Security",
         "url": "soap-client-security-inbound-security-config",
-        "verifyBuild": true,
+        "verifyBuild": false,
         "verifyOutput": false,
         "disablePlayground": true,
         "isLearnByExample": false
       },
       {
         "name": "Outbound Security",
         "url": "soap-client-security-outbound-security-config",
-        "verifyBuild": true,
+        "verifyBuild": false,
```

Comment:
```suggestion
        "verifyBuild": true,
```

Reply:
Fixed in fc222e13178db97b3f9a35effbb13b91845cad54